db/dcrpg/internal/blockstmts.go

```diff
@@ -101,7 +97,7 @@ const (
 	SelectBlockHashByHeight = `SELECT hash FROM blocks WHERE height = $1 AND is_mainchain = true;`
 	SelectBlockHeightByHash = `SELECT height FROM blocks WHERE hash = $1;`
 
-	RetrieveBestBlock          = `SELECT * FROM blocks ORDER BY height DESC LIMIT 0, 1;`
+	//RetrieveBestBlock          = `SELECT * FROM blocks ORDER BY height DESC LIMIT 0, 1;`
```

Comment:
Any reason to keep this around?

Reply:
Nope.  I meant to remove that after verifying it was unused.  Thanks.